pedpy/methods/flow_calculator.py

```diff
@@ -23,8 +23,8 @@ def compute_n_t(
         measurement_line (MeasurementLine): line for which n-t is computed
 
     Returns:
-        DataFrame containing the columns 'frame', 'Cumulative pedestrians',
-        and 'Time [s]' and DataFrame containing the columns 'ID', and 'frame'.
+        DataFrame containing the columns 'frame', 'cumulative_pedestrians',
+        and 'time' and DataFrame containing the columns 'ID', and 'frame'.
```

Comment:
This should be extended to indicate that this is seconds since frame 0 and the unit is seconds